cmd/litestream/replicate.go

```diff
@@ -43,6 +49,8 @@ func (c *ReplicateCommand) ParseFlags(ctx context.Context, args []string) (err e
 	fs := flag.NewFlagSet("litestream-replicate", flag.ContinueOnError)
 	execFlag := fs.String("exec", "", "execute subcommand")
 	configPath, noExpandEnv := registerConfigFlag(fs)
+	mcpFlag := fs.Bool("mcp", false, "Enable MCP server (agent mode)")
+	mcpPortFlag := fs.Int("mcp-port", 8080, "Port for MCP server (default 8080)")
```

Comment:
Can you remove these from the CLI? We put all the real configuration stuff in the `Config`.

Reply:
done